Cleanup the test_node file (#33)
SUSE/m8a-tests: README.md

* If you are using the "container" fixture, your test will auto generate the right tests for _all_ the versions of your language. This is auto loaded, and doesn't need anything from your side except using the keyword "container". See below for more details.
* If you want to _skip_ some of those tests, use the decorator named `restrict_to_version`, which accepts a list of strings of the versions for which to run the test. See below for more details.

### The container fixture

Container fixture contains the black magic to run command on all the different kind of nodes
per language.
If you need to run a test for certain versions of container only, three choices (by order of preference):

1. Use `restrict_to_version` to limit which containers your test applies to.
2. Create your own fixture
3. Modify the `container` fixture.
The container fixture automatically finds the testfile filename, uses it to guess the language of the container under test,
and starts all the necessary containers. See also conftest.py

### The restrict_to_version decorator

This decorator accepts a list of strings matching the versions from `data.py`.

To use it:

> 1. add `from conftest import restrict_to_version` in your imports.

Reviewer: ```suggestion
1. add `from conftest import restrict_to_version` to your imports.
```

> 2. wrap your code like this (assuming openjdk here):

Reviewer: ```suggestion
2. wrap your test function as follows (assuming openjdk here):
```

```Python
@restrict_to_version(['11'])
def mytest(container):
    pass
```

> This example above means that, from all the containers for openjdk, the `mytest` test will only run for openjdk '11' container.

Reviewer: ```suggestion
In the example above only the `openjdk:11` container will be used to run the test function `mytest`.
```
